optaplanner-quarkus-integration/optaplanner-quarkus/deployment/pom.xml

```diff
@@ -82,6 +88,12 @@
       <artifactId>drools-core-dynamic</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.yaml</groupId>
+      <artifactId>snakeyaml</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
@mareknovotny Would you might reverting the original commit completely, so that we don't end up with these useless exclusions after the PR si merged?

Reply:
reverted